examples/axios/cosmos.proxies.js

```diff
@@ -0,0 +1,3 @@
+import createXhrProxy from 'react-cosmos-xhr-proxy';
+
+export default [createXhrProxy()];
```

Comment:
Would it be too much trouble to support exporting a single proxy without wrapping it in an Array? I imagine someone will open an issue about trying that in the future.

Reply:
Should be fairly simple, something like `Array.isArray(proxies) ? proxies : [proxies]` somewhere in the codebase. My only worry if it doesn't add confusion when newcomes read the examples and in some places `cosmos.proxies.js` returns a list, while in others just a proxy. Maybe they'll think it's two different things. Also, my assumption is that any mature codebase will rely on more than one proxy. Any similar examples for other projects?

Reply:
Maybe it's just weird that all examples have only one proxy. You're right this is already a pretty advanced topic 👍

Reply:
Hmm, what about the [first example](https://github.com/react-cosmos/react-cosmos/tree/420-improved-proxy-api#where-to-put-proxies)?

Reply:
Rewrote the first example to have better annotations and to feature most popular proxies: Redux, React Router and Fetch.

Reply:
Also with a link to the proxy boilerplate.

![image](https://user-images.githubusercontent.com/250750/30134214-9f5c04fa-9356-11e7-8b55-bce682867e18.png)